To: vim_dev@googlegroups.com
Subject: Patch 7.3.351
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.351
Problem:    Text formatting uses start of insert position when it should not.
	    (Peter Wagenaar)
Solution:   Do not use Insstart when intentionally formatting.
Files:	    src/edit.c


*** ../vim-7.3.350/src/edit.c	2011-09-21 18:23:02.000000000 +0200
--- src/edit.c	2011-10-26 16:58:21.000000000 +0200
***************
*** 6078,6083 ****
--- 6078,6084 ----
  	 * Stop at first entered white when 'formatoptions' has 'v'
  	 */
  	while ((!fo_ins_blank && !has_format_option(FO_INS_VI))
+ 		    || (flags & INSCHAR_FORMAT)
  		    || curwin->w_cursor.lnum != Insstart.lnum
  		    || curwin->w_cursor.col >= Insstart.col)
  	{
*** ../vim-7.3.350/src/version.c	2011-10-26 13:19:23.000000000 +0200
--- src/version.c	2011-10-26 17:03:37.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     351,
  /**/

-- 
From "know your smileys":
 :-*	A big kiss!

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
